distr_tracing/distr_tracing_tempo/distr-tracing-tempo-installing.adoc

```diff
@@ -0,0 +1,36 @@
+:_content-type: ASSEMBLY
+[id="dist-tracing-tempo-installing"]
+= Installing distributed tracing platform Tempo
+include::_attributes/common-attributes.adoc[]
+:context: install-distributed-tracing
+
+toc::[]
+
+You can install {DTProductName} on {product-title} in either of two ways:
+
+* You can install {DTProductName} as part of {SMProductName}. Distributed tracing is included by default in the Service Mesh installation. To install {DTProductName} as part of a service mesh, follow the xref:../../service_mesh/v2x/preparing-ossm-installation.adoc#preparing-ossm-installation[Red Hat Service Mesh Installation] instructions. You must install {DTProductName} in the same namespace as your service mesh, that is, the `ServiceMeshControlPlane` and the {DTProductName} resources must be in the same namespace.
+
+* If you do not want to install a service mesh, you can use the {DTProductName} Operators to install {DTShortName} by itself. To install {DTProductName} without a service mesh, use the following instructions.
```

Comment:
Currently, this is not true because Tempo is not integrated with OSSM yet. There is only one way to install Tempo.